apps/api/src/app/environments/environments.controller.ts

```diff
@@ -152,23 +148,4 @@ export class EnvironmentsController {
 
     return await this.regenerateApiKeysUsecase.execute(command);
   }
-
-  @Put('/widget/settings')
```

Comment:
what happened to this?

Reply:
Me and @scopsy agree on removing this because it is not used anymore.